app/src/main/java/ar/com/wolox/android/example/ui/login/LoginFragment.kt

```diff
@@ -0,0 +1,44 @@
+package ar.com.wolox.android.example.ui.login
+
+import androidx.core.widget.addTextChangedListener
+import ar.com.wolox.android.R
+import ar.com.wolox.android.databinding.FragmentLoginBinding
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+
+class LoginFragment private constructor() : WolmoFragment<FragmentLoginBinding, LoginPresenter>() {
```

Comment:
Remember you have to implement the LoginView. 
You can implement classes in kotlin by adding after a comma:
`class LoginFragment : WolmoFragment<FragmentLoginBinding, LoginPresenter>(), LoginView`